### ISSUE TITLE
On iOS, open Google Maps if present, else open Apple Maps

### ISSUE THREAD

**Issue by arhmnsh** [brh55/react-native-open-maps#25]
Is it possible to open Google Maps if it is installed, else open Apple Maps?

**Comment by brh55** [comment]
I completely agree with you! That would definitely improve the user experience if we only open apps available. Unfortunately, it's not currently implemented, but implementation would involve something like this:
https://stackoverflow.com/questions/47652375/check-if-google-maps-app-is-installed-in-react-native-ios

Afterward, the developer using the package would just need to add the URL schemes in their app configuration if you want to ensure this package can fallback on an app level.

If you feel up for the task, please give it a whirl! I'm a bit busy these days with personal things.

**Comment by nwaughachukwuma** [comment]
@brh55 and @arhmnsh what's the status of this? I can make a PR with this addition. What do you think?

**Comment by nwaughachukwuma** [comment]
@brh55 a PR has been submitted

**Comment by nwaughachukwuma** [comment]
Anyone looking for a solution around this can use this [fork](https://github.com/nwaughachukwuma/react-native-open-maps.git)

**Comment by daveyjones** [comment]
This is exactly the behavior I was looking for.

**Comment by lucksp** [comment]
I too am looking for this behavior
